src/stellar/stellarEngine.ts

```diff
@@ -296,8 +295,8 @@ export class StellarEngine extends CurrencyEngine<StellarPlugin> {
           currencyCode = bal.asset_type
         }
         const denom = getDenomInfo(this.currencyInfo, currencyCode)
-        if (denom && denom.multiplier) {
-          const nativeAmount = bns.mul(bal.balance, denom.multiplier)
+        if (denom != null && denom.multiplier) {
```

Comment:
missed sketchy null

Reply:
The goal of this PR is not to fix all the possible TS errors but to just not be "worse than Flow". A future commit auto adds the appropriate eslint exception comment which we can remove in future work and slowly clean up. Otherwise you'd be making 100s of these type of comments